apiserver/plane/app/serializers/workspace.py

```
@@ -11,7 +11,8 @@
     WorkspaceMemberInvite,
     WorkspaceTheme,
     WorkspaceUserProperties,
-    WorkspaceUserLink
+    WorkspaceUserLink,
+    WorkspaceHomePreference
```

Ruff (0.8.2)

`plane.db.models.WorkspaceHomePreference` imported but unused

Remove unused import: `plane.db.models.WorkspaceHomePreference`

(F401)